my-cart.html

```diff
@@ -0,0 +1,249 @@
+              <div class="card-price">Rs 5799</div>
+            </div>
+
+            <div class="display-flex mt-1">
+              <button class="border-1px-circle-btn">
+                <i class="fas fa-minus"></i>
+              </button>
+              <div
+                class="border-1px-square cart-qty text-center text-bold font-large"
+              >
+                5
+              </div>
+              <button class="border-1px-circle-btn">
+                <i class="fa fa-plus"></i>
+              </button>
+            </div>
+
+            <div class="row">
+              <button class="btn secondary-btn">Move to wishlist</button>
+              <button class="btn primary-btn-text-icon">
+                <i class="fas fa-trash-alt"></i>
+              </button>
+            </div>
+          </div>
+        </div>
+
+        <div class="card-horizontal cart-item">
+          <div class="card-img">
+            <img
+              loading="lazy"
+              src="./assets/product-image3-landscape.jpeg"
+              alt="Nature"
+              class="img-responsive"
+            />
+          </div>
+
+          <div class="card-details">
+            <div class="card-description">
+              <p class="text-medium text-semibold">
+                Sleeveless Colorblock Men Quilted Jacket .
+              </p>
+              <div class="card-price">Rs 5799</div>
+            </div>
+
+            <div class="display-flex mt-1">
+              <button class="border-1px-circle-btn">
+                <i class="fas fa-minus"></i>
+              </button>
+              <div
+                class="border-1px-square cart-qty text-center text-bold font-large"
+              >
+                5
+              </div>
+              <button class="border-1px-circle-btn">
+                <i class="fa fa-plus"></i>
+              </button>
+            </div>
+
+            <div class="row">
+              <button class="btn secondary-btn">Move to wishlist</button>
+              <button class="btn primary-btn-text-icon">
+                <i class="fas fa-trash-alt"></i>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="cart-price-detail display-flex-column">
+        <button class="btn secondary-btn-outline">
+          <span class="btn-icon"><i class="fas fa-tag"></i></span>&nbsp&nbsp
+          Apply Coupon
+        </button>
+        <h5 class="text-bold">PRICE DETAILS (2 items)</h5>
+        <hr />
+        <div class="row text-semibold">
+          <div class="w-70">Total Price</div>
+          <div class="w-30">₹ 4999</div>
+        </div>
+
+        <div class="row text-semibold">
+          <div class="w-70">Discount</div>
+          <div class="w-30">-₹ 1999</div>
+        </div>
+
+        <div class="row text-semibold">
+          <div class="w-70">Delivery Charges</div>
+          <div class="w-30">₹ 499</div>
+        </div>
+
+        <hr />
```

Comment:
suggesstion: instead of hr use border property for styling which is more flexible

Reply:
Ok i will do